versions.tf

```
@@ -3,7 +3,7 @@
 # ----------------------------------------------------------------------------------------------------------------------
 
 terraform {
-  required_version = ">= 0.12.20, < 0.16.0"
+  required_version = ">= 0.12.20, < 1.0.6"
```

Hey, thanks for your contribution.. we can assume that the module will support all major 1.x releases if hashicorp will stick to semantic versioning in their major releases.

```suggestion
  required_version = ">= 0.12.20, < 2.0"
```

for a release we would also need to update tests to run with the new version and ofc update docs ;) but we can already merge this once the version constraint is fixed from my point of view.

cc @soerenmartius

---

makes sense.